Connection.php

```diff
@@ -288,7 +288,7 @@ public function open()
      */
     public function close()
     {
-        if ($this->_socket !== false) {
+        if ($this->_socket !== null) {
```

Comment:
please revert these changes. checking for `false` is correct.

Reply:
*/
public function close()
{
   if($this->_socket !==null){
  if($this->_socket !==false){
}